_activities/visualization.md

```diff
@@ -6,11 +6,11 @@ redirect_from:
   - /workinggroups/visualization.html
 ---
 
-The Visualisation Activity Area and Working Group (WG) gathers experts, developers, users and all people from the Particle Physics community interested in interactive data visualization and event displays. 
+The Visualisation Activity Area gathers experts, developers, users and all people from the Particle Physics community interested in interactive data visualization and event displays. 
```

Comment:
Oops, we missed a "zation" now that we're using "sation" everywhere else :S.

Reply:
Indeed, pages written by a mish-mash of people with different spelling norms...